src/storage/storage.ts

```diff
@@ -78,6 +84,27 @@ export class Storage implements IStorage {
     this.version = version;
   }
 
+  /**
+   * Set the tx signing function
+   * @param {EcdsaTxSign} txSign The signing function
+   */
+  setTxSign(txSign: EcdsaTxSign): void {
+    this.txSignFunc = txSign;
+  }
+
+  /**
+   * Sign the transaction
+   * @param {Transaction} tx The transaction to sign
+   * @param {string} pinCode The pin code
+   * @returns {Promise<IInputSignature[]>} The signatures
+   */
+  async signTx(tx: Transaction, pinCode: string): Promise<IInputSignature[]> {
```

Comment:
suggestion: This method does not sign the transaction, but only returns an array of signatures for another method to inject it into the `Transaction` object.

I'd suggest renaming this method to `getTxSignatures(tx, pinCode)` to better reflect that it is a pure function.

Reply:
Done